cmd/kubeadm/app/cmd/join.go

```diff
@@ -361,10 +390,6 @@ func newJoinData(cmd *cobra.Command, args []string, opt *joinOptions, out io.Wri
 		}
 	}
 
-	if err := validation.ValidateMixedArguments(cmd.Flags()); err != nil {
```

Comment:
I don't think we need to remove this.
Removing it would allow for `--advertise-address` and `--config` to be combined. However, the value passed to `--advertise-address` will be replaced with the contents of the config passed through a file.

Reply:
it's being moved, on the top of the function to perform the validation as early as possible.

Reply:
Ah, must have missed that somehow.

/lgtm